app/supplejack/extraction/execution.rb

```diff
@@ -40,6 +35,14 @@ def call
 
     private
 
+    def extract_and_save_document(request)
+      @de = DocumentExtraction.new(request, @extraction_job.extraction_folder, @previous_request&.body)
+      @previous_request = @de.extract
+      @de.save
```

Comment:
do we need to check if this was successful?

Reply:
Na it's up to the harvester to decide if something has gone wrong with the extraction and fix the issue themselves.